openshift-container-platform-4/policies/CA-Security_Assessment_and_Authorization/component.yaml

```diff
@@ -9,68 +9,293 @@
 
 - control_key: CA-7
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - key: a
+      text: |
+        Engineering progress can be tracked at:
+
+        https://jira.coreos.com/browse/CMP-255
+    - key: b
+      text: |
+        Engineering progress can be tracked at:
+
+        https://jira.coreos.com/browse/CMP-255
+    - key: c
+      text: |
+        Engineering progress can be tracked at:
+
+        https://jira.coreos.com/browse/CMP-255
+    - key: d
+      text: |
+        Engineering progress can be tracked at:
+
+        https://jira.coreos.com/browse/CMP-255
+    - key: e
+      text: |
+        Engineering progress can be tracked at:
+
+        https://jira.coreos.com/browse/CMP-255
+    - key: f
+      text: |
+        Engineering progress can be tracked at:
+
+        https://jira.coreos.com/browse/CMP-255
+    - key: g
+      text: |
+        Engineering progress can be tracked at:
+
+        https://jira.coreos.com/browse/CMP-255
+
+- control_key: CA-7 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        Employment of assessors or assessment teams is outside the
+        scope of OpenShift configuration guidance.
+
+- control_key: CA-7 (2)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control was withdrawn by NIST.
+
+- control_key: CA-7 (3)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        Organizational trend analyses processes are
+        outside the scope of OpenShift configuration guidance.
+
+- control_key: CA-8
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
     - text: |
-        'This control reflects organizational procedures/policies, and is not
-        applicable to the configuration of OpenShift.'
+        Penetration testing is outside the scope of
+        OpenShift configuration guidance.
+
+- control_key: CA-8 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        Employment of independent penetration testers
+        is out of scope for OpenShift configuration
+        guidance.
+
+- control_key: CA-8 (2)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        Red Team Exercises are out of scope for
+        OpenShift configuration guidance.
 
 - control_key: CA-9
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: partial
+  narrative:
+    - key: a
+      text: |
+        Engineering progress can be tracked at:
+
+        https://jira.coreos.com/browse/CMP-256
+    - key: b
+      text: |
+        Engineering progress can be tracked at:
+
+        https://jira.coreos.com/browse/CMP-256
```

Comment:
Should this be ``partial`` (e.g. we have some answer, but have gaps) or ``planned`` (eg product deficiency is being tracked?)

Reply:
@shawndwells yes this should be `partial`